src/routes/SelectRepos/reducer.js

```diff
@@ -127,6 +127,12 @@ export default function(state = initialState, action) {
         githubError: null,
       };
     }
+    case types.RESET_SELECTED_REPOS: {
```

Comment:
This needs a test in the reducer.test file

Reply:
😕